rubygems/rubygems.org
# add mastodon icon

### Bug: 
https://github.com/rubygems/rubygems.org/issues/4600 
### Goal 
Add mastodon icon to email footer 
### How to test:
Run rails server
Open browser and update link to [LOCAL HOST]/rails/mailers
Click any of the links listed to see the footer
### Before 
<img width="418" alt="Screenshot 2024-06-13 at 3 23 07 PM" src="https://github.com/rubygems/rubygems.org/assets/29336370/c169a570-e9d7-41ce-ac03-e1aa1e7a5337">

### After 
<img width="925" alt="Screenshot 2024-06-13 at 3 20 59 PM" src="https://github.com/rubygems/rubygems.org/assets/29336370/2f6e371a-6086-4ff3-add6-7ef43d01c205">
